src/test/resources/com/puppycrawl/tools/checkstyle/checks/javadoc/abstractjavadoc/InputAbstractJavadocPositionWithSinglelineComments.java

```diff
@@ -11,16 +11,16 @@
 class/**nope*/ V/**nope*/{
 	/**Javadoc*/ //noise
 	protected/**nope*/ class/**nope*/ N/**nope*/{/**nope*/}
-	
+
```

Comment:
Please move update for spaces in separate PR , I will merge it quickly.

This PR is going to be complicated, lets minimize unrelated changes.